lib/zendesk_apps_support/assets/installed.js.erb

```diff
@@ -0,0 +1,15 @@
+(function() {
+  <% appsjs.each do |appjs| %>
+    <%= appjs %>
+  <% end %>
+
+  <% installations.each do |installation| %>
+    ZendeskApps[<%= installation.app.name.to_json %>].install(<%= installation.to_json %>);
+  <% end %>
+
+  <% installation_orders.each do |loc_name, installation_list| %>
+    ZendeskApps.sortAppsForSite(<%= loc_name.to_json %>, <%= installation_list.to_json %>);
+  <% end %>
+}());
+
+ZendeskApps.trigger && ZendeskApps.trigger('ready');
```

Comment:
I think if we expect it to be an event emitter we should make it an event emitter in ZAF, rather than expecting the host to make it one. In Zopim we were actually using a callback queue: https://github.com/zopim/meshim-frontend/pull/1323/files#diff-f6efbb95e2a272f2ae751763b5547728R77